tests/vmi_hook_sidecar_test.go

```diff
@@ -213,8 +220,26 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 			})
 		})
 
+		Context("with ConfigMap in sidecar hook annotation", func() {
+			BeforeEach(func() {
+				vmi.ObjectMeta.Annotations = RenderSidecarWithConfigMap(hooksv1alpha2.Version)
+			})
+
+			It("should update domain XML with SM BIOS properties", func() {
+				_, err = virtClient.CoreV1().ConfigMaps(testsuite.GetTestNamespace(vmi)).Create(context.TODO(), RenderConfigMap(), metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				vmi = tests.RunVMIAndExpectLaunch(vmi, <timeout>)
```

Comment:
You need to set a proper timeout here.

Reply:
That was embarassing. Thanks for catching. 🤦🏽‍♂️